backend/src/data/models/course-category/course-category.model.ts

```diff
@@ -0,0 +1,13 @@
+import { DbTableName } from '~/common/enums/enums';
+import { Abstract } from '../abstract/abstract.model';
+
+class CourseCategory extends Abstract {
+  'name': string;
+  'key': string;
```

Comment:
```suggestion
  'name': string;
  
  'key': string;
```

Reply:
Do I leave the model file, or should I remove it for now?

Reply:
just add spaces between keys